LoopBackTests/bin/run-unittests

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -ex
+trap 'kill $(jobs -p)' EXIT
+
+# record the build environment
+xcodebuild -version
+xcodebuild -showsdks
+instruments -s devices
+
+# build the target
+xcodebuild \
+    -project LoopBack.xcodeproj \
```

Comment:
@rmg thank you for the review and the suggestion. Sounds good. I've added `$XCODEBUILD_ARGS` to the `xcodebuild` command line.  Does this look good?